README-dev.md

```diff
@@ -141,8 +141,23 @@ npm run test
 npm run test:unit
 ```
 
+#### Running Mina-Signer Tests
```

Comment:
So there are a few ways you could go about writing this into the readme-dev. Essentially it's the fact that

`npm run build` 

is not sufficient, and makes rise 140 above incorrect. `npm run test` will fail once it gets through the `/lib` tests and arrives at the `/mina-signer` tests. as @mitschabaude pointed out, the `ci` moves into `/mina-signer` and runs build again before running `npm run test`. 

perhaps this change should be behind a header for running mina-signer tests and should simply be at the topmost reference to building. it's not as svelte however

 ```
npm run build
cd src/mina-signer
npm run build
cd ../..
```

so i stuck it here. 🤷‍♂️

Reply:
That works well!